src/class-wp-sentry-php-tracker.php

```diff
@@ -125,11 +143,18 @@ public function get_client(): HubInterface {
 	 * @return array
 	 */
 	public function get_default_tags(): array {
-		return [
-			'wordpress' => get_bloginfo( 'version' ),
-			'language'  => get_bloginfo( 'language' ),
-			'php'       => phpversion(),
```

Comment:
In case someone is wondering, `php` is already added by the SDK as `runtime` so this was kind of duplicated data. (I'm assuming that's the case why it's removed now.)

Reply:
Yeah exactly, I noticed that too, and the SDK does it much cleaner already IMHO :)